documenation.markdown

```diff
@@ -148,7 +146,7 @@ read the User Guide!
 
     $ cylc graph test1 &
 
-![img/cylc-graph.png](img/cylc-graph.png)
+<img class="cylc-graph" src="/assets/img/cylc-graph.png" alt="Cylc Graph" />
```

Comment:
This doesn't display.  Needs the markdown form with `relative_link` filter as above?